roles/ceph-validate/tasks/check_system.yml

```diff
@@ -46,12 +46,12 @@
     - ceph_repository == 'uca'
     - ansible_distribution != 'Ubuntu'
 
-- name: fail on unsupported openSUSE distribution
+- name: "fail on unsupported openSUSE distribution (42.x)"
```

Comment:
Maybe a better way to do it is to compare to the major distribution version fact

```console
$ ansible -i hosts all -m setup -a 'filter=ansible_distribution_major_version'
xxxxxxxxx | SUCCESS => {
    "ansible_facts": {
        "ansible_distribution_major_version": "15",
        "discovered_interpreter_python": "/usr/bin/python3.6"
    },
    "changed": false
}
```

So something like `ansible_distribution_major_version != '15'` should be enough to support Leap 15.x

Reply:
Do you want to me explicitly whitelist Leap 15.x only, so Leap 16.x (once released) will be unsupported? Or do you want me to only blacklist 42.x?

Reply:
For now we can focus on Leap 15 because we don't really now what will be the next openSuSe release since they changed a lot the release number during last years.